game.py

```diff
@@ -12,15 +12,26 @@
     '-----------'
 ]
 
-
 def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and not is_word_guessed(snowman_word, correct_letter_guess_statuses):
```

Comment:
Nice work with this `while` loop! We don't actually know how many times it will take a user to guess the word or hit the limit of wrong guesses (could keep make invalid guesses) so this the perfect choice for that situation! 

We could also rework your loop to depend on just the wrong number of guesses or if the word was guessed. We could treat the other condition like a flag and exit out of our function early if said condition is met. These changes would look a little something like this:

```python 

while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES: 
    ...

    if is_word_guessed: 
        print(f"Congratulations, you win!")
        return
``` 
With the `return` keyword not only do we exit out of the loop but also the function.